app/scripts/modules/kubernetes/src/v2/kubernetes.v2.module.ts

```diff
@@ -104,6 +109,7 @@ module(KUBERNETES_V2_MODULE, [
       detailsController: 'kubernetesV2ServerGroupDetailsCtrl',
       detailsTemplateUrl: require('./serverGroup/details/details.html'),
       transformer: 'kubernetesV2ServerGroupTransformer',
+      infra: SETTINGS.adHocInfraWrites,
```

Comment:
Why are we adding this `infra` property to each type of entity rather than just setting it as a top-level property of the provider? Also can we be a bit more descriptive than just `infra`?

Reply:
Fixed